salt/modules/file.py

```diff
@@ -1589,6 +1590,13 @@ def _get_line_indent(src, line, indent):
 
     return ''.join(idt) + line.strip()
 
+def _get_line_ending(src, line):
```

Comment:
This seems to add line ending, rather than get_line_ending consider a different name for the function.
e.g. _add_src_eol(src, line).

Reply:
@damon-atkins I wanted to be consistent with name scheme of _get_line_indent, which ads indentation to line.

Reply:
Good idea, change it